src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -713,6 +721,27 @@ public boolean isAllFeedbackPathOptionsEnabled() {
         return true;
     }
     
+    public boolean isAllRecipientOptionsDisplayed(int questionNumber) {
+        List<WebElement> recipientOptions =
+                browser.driver.findElements(By.cssSelector("#recipienttype-" + questionNumber + " option"));
+        for (WebElement recipientOption : recipientOptions) {
+            String recipientOptionClass = recipientOption.getAttribute("class");
+            if ("hidden".equals(recipientOptionClass)) {
```

Comment:
Is `.isDisplayed()` still not working? It's still as fragile as before, to things like `class="hidden class1 class2"`.